core/domain/exp_domain_test.py

```diff
@@ -7376,7 +7503,7 @@ class HTMLMigrationUnitTests(test_utils.GenericTestBase):
 """)
 
 # pylint: disable=line-too-long
-    YAML_CONTENT_V37_IMAGE_DIMENSIONS = ("""author_notes: ''
```

Comment:
Is this really supposed to be renamed and not replaced with a new one? Or does this need to be edited at all? Since in file, I see a lot of old versions as well.

Reply:
@aks681 I used #8638 as a reference for my PR. Since it was renamed there I also did the same.